dhctl/pkg/kubernetes/actions/manifests/manifests.go

```diff
@@ -69,189 +73,196 @@ func GetDeckhouseDeployTime(deployment *appsv1.Deployment) time.Time {
 }
 
 func ParametrizeDeckhouseDeployment(deployment *appsv1.Deployment, params DeckhouseDeploymentParams) *appsv1.Deployment {
```

Comment:
Please fix typo to `ParametrizeDeckhouseDeployment`

Reply:
Yeah, for sure. I completely overlooked it.